src/repository/fetchJson.js

```diff
@@ -0,0 +1,13 @@
+import fetch from 'node-fetch';
+
+const fetchJson = url => fetch(url)
```

Comment:
To też jest jakieś podejście ;) niech zostanie tak jak jest.
Z mojego doświadczenia przy fetchowaniu prędzej czy później dojdzie więcej kodu typu: rzucanie wyjątków, ściąganie informacji z headerów czy tez obsługa statusów innych niż 200.